Support for Raspberry PI

I'm trying to set up sampler to work as a dashboard running on a Raspberry PI.
I managed to get it to compile on the PI but when i try to run it i get the following error:
```
# sampler --config conf.yml
panic: non-positive interval for NewTicker [recovered]
        panic: non-positive interval for NewTicker

goroutine 1 [running]:
main.handleCrash(0x108e5a0, 0x10708a0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1075cc0)
        /root/sampler-1.0.3/main.go:127 +0x170
panic(0x339418, 0x100e198)
        /usr/lib/go-1.11/src/runtime/panic.go:513 +0x194
time.NewTicker(0xd964b800, 0xffffffff, 0x123a5a0)
        /usr/lib/go-1.11/src/time/tick.go:23 +0x1bc
github.com/sqshq/sampler/data.NewSampler(0x1085310, 0x1085320, 0x3, 0x4, 0x650634, 0x0, 0x0, 0x10708a0, 0x0, 0x0, ...)
        /root/sampler-1.0.3/data/sampler.go:20 +0x2c
main.(*Starter).start(0x10d7f30, 0x3eb490, 0x1183b80, 0x1085310, 0x107ad70, 0xb, 0x10759c0, 0x2, 0x2, 0x107bd88, ...)
        /root/sampler-1.0.3/main.go:68 +0x288
main.(*Starter).startAll(0x10d7f30, 0x1202000, 0x1196060, 0x1196090)
        /root/sampler-1.0.3/main.go:49 +0x538
main.main()
        /root/sampler-1.0.3/main.go:115 +0x26c
```

--- a/data/sampler.go
+++ b/data/sampler.go
@@ -17,7 +17,7 @@ type Sampler struct {
 
 func NewSampler(consumer *Consumer, items []*Item, triggers []*Trigger, options config.Options, fileVariables map[string]string, rateMs int) *Sampler {
 
-	ticker := time.NewTicker(time.Duration(rateMs * int(time.Millisecond)))
+	ticker := time.NewTicker(time.Duration(uint32(rateMs) * uint32(time.Millisecond)))
 
 	sampler := &Sampler{
 		consumer,

